distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -491,7 +388,7 @@ type Language
    Arguments:
    - value: The value to check.
 is_atom_constructor : Any -> Boolean
-is_atom_constructor value = @Builtin_Method "Meta.is_atom_constructor"
+is_atom_constructor value = @Tail_Call Meta_Helpers.is_atom_constructor_builtin value
```

Comment:
This could be rewritten as `Meta.meta value . is_a Meta.Atom`, but I guess that's too complicated? Or isn't it?

Reply:
No, `is_atom_constructor` doesn't seem to have any usages, but for example `is_atom` is tested:

![Usages of is_atom](https://github.com/user-attachments/assets/a5eaa5bd-c5f8-487b-ab7b-5e545500d4e3)

Reply:
At the end I kept all the existing `Meta.is_` checks. I only encapsulated them all into a single builtin in 8c6ddd6da459249cdb7b7366fa2192465c39dda2